### PR TITLE
Pages & content

### PR DESCRIPTION
**Pull request by Benson-J**
Added content within the main tags & styled it.

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -22,8 +22,51 @@
             <a href="#"><i class="fab fa-twitter-square"></i></a>
         </nav>
     </header>
-    <main>
-        Portfolio
+    <main class="portfolioMain">
+        <div class="mainContent">
+            <div class="portfolioBox">
+                <img src="portfolioScreenShot.png">
```
Comment (imogenhallett) [inline portfolio.html]: img does not have an alt tag, consider putting an alt tag

File: portfolio.html [hunk 2]
```diff
@@ -22,8 +22,51 @@
...
+                <div class="portfolioBoxText">
+                    <h3>Portfolio Website</h3>
+                    <p>The website you're currently on, & my first Mayden Academy project.</p>
+                </div>
+            </div>
+            <div class="portfolioBox">
+                <img src="placeholderScreenshot.png">
```
Comment (imogenhallett) [inline portfolio.html]: img does not have an alt tag, consider putting an alt tag

File: portfolio.html [hunk 3]
```diff
@@ -22,8 +22,51 @@
...
+                <div class="portfolioBoxText">
+                    <h3>Placeholder</h3>
+                    <p>A placeholder to demonstrate box resizing & wrapping.</p>
+                </div>
+            </div>
+            <div class="portfolioBox">
+                <img src="placeholderScreenshot.png">
```
Comment (imogenhallett) [inline portfolio.html]: img does not have an alt tag, consider putting an alt tag

File: portfolio.html [hunk 4]
```diff
@@ -22,8 +22,51 @@
...
```
Comment (imogenhallett) [inline portfolio.html]: img does not have an alt tag, consider putting an alt tag

File: portfolio.html [hunk 5]
```diff
@@ -22,8 +22,51 @@
...
```
Comment (imogenhallett) [inline portfolio.html]: img does not have an alt tag, consider putting an alt tag

File: portfolio.html [hunk 6]
```diff
@@ -22,8 +22,51 @@
...
```
Comment (imogenhallett) [inline portfolio.html]: img does not have an alt tag, consider putting an alt tag

**Review by imogenhallett — COMMENTED** [review]
Really nice and clean code :-)

File: styles.css [hunk 7]
```diff
@@ -87,35 +88,23 @@ main {
...
-    .socialLinks {
-        display: none;
-    }
-    .topNav a {
-        margin: 0;
-    }
+.mainContent {
+    background-color: rgba(0,0,0,0.3);
+    border-radius: 20px;
+    max-width:100%;
```
Comment (mporam) [inline styles.css]: formatting. space after colon

File: styles.css [hunk 8]
```diff
@@ -127,15 +116,139 @@ h1 {
...
+
+.portfolioBox {
+    max-width: 30%;
+    min-width: 400px;
+    position: relative;
+    margin: 10px;
+}
+
+.portfolioBoxText {
+    display: block;
```
Comment (mporam) [inline styles.css]: does nothing. divs are block by default

File: about.html [hunk 9]
```diff
@@ -22,19 +22,40 @@
...
+        <div class="mainContent">
+            <h1>I'm a full-stack web development student at Mayden Academy.</h1>
+            <section class="aboutWrapper">
+                <article class="experience">
+                    <h2>Experience</h2>
+                    <ul class="aboutTopList">
+                        <li>Mayden Academy, 08/18-12/18</li>
+                    </ul>
+                </article>
+                <div class="borderLine"></div>
```
Comment (mporam) [inline about.html]: put a border on the article tag, instead of using an empty div

**Review by mporam — CHANGES_REQUESTED** [review]

File: styles.css [hunk 10]
```diff
@@ -132,8 +132,8 @@ h3 {
     justify-content: space-between;
 }
 
-.aboutWrapper div {
-    width: 50%;
+.aboutWrapper article{
```
Comment (mporam) [inline styles.css]: space before bracket

File: styles.css [hunk 11]
```diff
@@ -144,19 +144,16 @@ ul {
...
+    border: solid #ffffff;
+    border-width: 0 1px 0 0;
 }
 
-.borderLine {
-    max-width: 0;
+.skills {
+    text-align: center;
     border: solid #ffffff;
     border-width: 0 1px 0 0;
```
Comment (mporam) [inline styles.css]: you can do this with one property:
`border-right: 1px solid #ffffff;`

**Review by mporam — CHANGES_REQUESTED** [review]

**Review by mporam — APPROVED** [review]

**Pull request merged**